src/main/java/frc/robot/commands/ShootBallCommand.java

```diff
@@ -2,6 +2,4 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-public class ShootBallCommand extends CommandBase {
-
-}
+public class ShootBallCommand extends CommandBase {}
```

Comment:
Try to not mix formatting changes with actual changes. It clutters up the PR.  You can choose only specific files during add, and if you need to choose specific parts of those files, do `git add -i` for an interactive staging